rules/reducing-design-debt/rule.md

```diff
@@ -0,0 +1,111 @@
+---
+type: rule
+tips: ""
+title: "Do you know how to avoid design debt? "
+seoDescription: Making ad-hoc UI changes without a designer leads to design
+  debt. Learn how to prevent inconsistencies and poor UX by following a better
+  workflow.
+uri: avoid-design-debt
+authors:
+  - title: ""
+created: 2025-04-23T11:45:00.000Z
+archivedreason: ""
```

Comment:
Hi @micsblank 

Can you add some related rules?
I thought about the "less is more", the "designers test please" and "tech debt"